hazelcast/src/main/java/com/hazelcast/sql/impl/security/SqlSecurityContextImpl.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql.impl.security;
+
+import com.hazelcast.security.SecurityContext;
+
+import javax.security.auth.Subject;
+import java.security.Permission;
+
+public class SqlSecurityContextImpl implements SqlSecurityContext {
```

Comment:
Wouldn't it be cleaner to put this class to the EE repo and make it available through some new method in the `SecurityContext` class? E.g. `SecurityContext.createSqlSecurityContext(Subject)`

Reply:
Agree. Moved `SqlSecurityContextImpl` to enterprise. Now it is created from the `SecurityContextImpl`.